_data/assetPaths.json

```diff
@@ -1,9 +1,9 @@
 {
-  "admin.js": "/assets/js/admin-44W43542.js",
-  "admin.map": "/assets/js/admin-44W43542.js.map",
-  "app.js": "/assets/js/app-XWR645AF.js",
-  "app.map": "/assets/js/app-XWR645AF.js.map",
+  "admin.js": "/assets/js/admin-OYJBR6FH.js",
```

Comment:
why are these asset shas hardcoded?

Reply:
This should be `.gitignored` but there maybe a problem on initial build without the file.